easybuild/easyconfigs/y/YaHS/YaHS-1.2.2-foss-2024a.eb

```diff
@@ -0,0 +1,27 @@
+# This file is an EasyBuild reciPY as per https://easybuild.io/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'MakeCp'
+
+name = "YaHS"
+version = "1.2.2"
+
+homepage = 'https://github.com/c-zhou/yahs'
+description = "Yet another Hi-C scaffolding tool"
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+source_urls = ['https://github.com/c-zhou/yahs/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9ee37f72ee9f62015fe92029d0fa97eec90963ddf15a2f4b760b45ee2e0014aa']
+
+files_to_copy = [(["yahs", "juicer", "agp_to_fasta"], "bin"), "LICENSE", "README.md"]
+
+sanity_check_paths = {
+    'files': ["bin/yahs", "bin/juicer", "bin/agp_to_fasta"],
+    'dirs': [],
+}
```

Comment:
can you add some `sanity_check_commands` here? for example
```python
sanity_check_commands = ['yahs -?']
```

Reply:
done